lib/GlobalStylesheets.js

```diff
@@ -30,13 +30,66 @@ function addStyle(css){
 
 function createStylesheet(stylesheet) {
   var style = assign({}, stylesheet.style);
+  var hoverStyles = {};
+  var activeStyles = {};
+  var focusStyles = {};
+
+  Object.keys(style).forEach(function(key) {
+    if (key.indexOf('hover') === 0) {
+      // Strip the key of the pseudo selector prefix and transform first char
+      // to lowercase to ensure correct markup generation and autoprefixing
+      var prop = key.substr(5);
```

Comment:
If we really wanted to work on perf, I'd suggest that we codegen a function that does this transformation without doing a bunch of substr operations. But that's probably overkill for now :)